client/packages.go

```diff
@@ -112,6 +114,42 @@ func (client *Client) List(names []string) ([]*Snap, error) {
 	return result, nil
 }
 
+func setPrice(snap *Snap, currency string) {
```

Comment:
I see. So the question is really whether it makes more sense to have the default price or the default currency in the struct, and given that there seems to be no further use of the default currency other than setting the price, maybe it's simpler.

Question for you then: would having the default currency make some UI easier?